week-3/implement/get-angle-type.js

```diff
@@ -21,3 +21,30 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+// Function to identify the type of angle
+//...............................................
+
+//Answer
+
+
+function getAngleType(angle) {
```

Comment:
great solution, you should take care of negative number, 360 and numbers greater than 360